components/setting_picture.tsx

```diff
@@ -15,35 +14,48 @@ import OverlayTrigger from 'components/overlay_trigger';
 import Tooltip from 'components/tooltip';
 import LoadingWrapper from 'components/widgets/loading/loading_wrapper';
 
-export default class SettingPicture extends Component {
+type Props = {
+    clientError?: ReactNode;
+    serverError?: ReactNode;
+    src?: string | null;
+    defaultImageSrc?: string;
+    file?: File | null;
+    loadingPicture?: boolean;
+    submitActive?: boolean;
+    onRemove?: () => void;
+    onSetDefault?: (() => Promise<void>) | null;
+    onSubmit?: (() => void) | null;
```

Comment:
similar to above comment

Reply:
Yup, user_settings_general can technically pass in a null value here